packages/network-enablement-controller/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Add Tron network support ([#6734](https://github.com/MetaMask/core/pull/6734))
```

Comment:
If poss, can you expand on the dev changes.
- Adding the Tron namespace to the enabledNetworkMap state.
- Also adding the keyring API to reuse types?